build.gradle

```diff
@@ -45,7 +45,7 @@ task clean(type: Delete) {
 //
 // It must be false for fat aar generation of MuxKalturaSDK to work
 // This is a bug/problem with the fat aar mechanism
-def inLocalDevModeFlag = false
```

Comment:
Yes/no?

I hope those stats do show we can get rid of fat-aars.

Reply:
I flipped it back but we need the java 11 update before we can see if it worked 🤞